gallery/src/Application.tsx

```diff
@@ -114,6 +121,24 @@ function Application() {
             <NxGlobalHeaderEmptyExample />
           </SectionScrollingWrapper>
         </Route>
+        <Route exact path="/NxToastSimpleLayoutExample">
+          <SectionScrollingWrapper>
+            <NxToastSimpleLayoutExample />
+          </SectionScrollingWrapper>
+        </Route>
+        <Route exact path="/NxToastComplexLayoutExample">
+          <SectionScrollingWrapper>
+            <NxToastComplexLayoutExample />
+          </SectionScrollingWrapper>
+        </Route>
+        <Route exact path="/NxToastLegacySectionScrollingExample">
+          <SectionScrollingWrapper>
+            <NxToastLegacySectionScrollingExample />
```

Comment:
Is there any actual difference between `NxToastLegacySectionScrollingExample` and `NxToastLegacyPageScrollingExample`? Usually when we have to show the two modes, the examples themselves are identical (and in fact there is only one codebase for both of them) and the only difference is the `SectionScrollingWrapper`.

Reply:
> Is there any actual difference between `NxToastLegacySectionScrollingExample` and `NxToastLegacyPageScrollingExample`? Usually when we have to show the two modes, the examples themselves are identical (and in fact there is only one codebase for both of them) and the only difference is the `SectionScrollingWrapper`.

Thanks for the heads up on this. I removed the second example, renamed the SectionScrolling example for more appropriate description, and updated Application.tsx here: [f025b31](https://github.com/sonatype/sonatype-react-shared-components/pull/688/commits/f025b31c06cb0d0a59a900dc14be67389b6a8aee).  Should there be a link to refer to legacy layouts and their different scrolling behaviour, or is it unnecessary?